src/front-end/typescript/lib/app/types.ts

```diff
@@ -71,25 +88,40 @@ export interface State {
     orgCreate?: Immutable<PageOrgCreate.State>;
     orgList?: Immutable<PageOrgList.State>;
     orgEdit?: Immutable<PageOrgEdit.State>;
+    cwuOpportunityCreate?: Immutable<CwuOpportunityCreate.State>;
+    cwuOpportunityEdit?: Immutable<CwuOpportunityEdit.State>;
+    cwuOpportunityView?: Immutable<CwuOpportunityView.State>;
+    cwuProposalCreate?: Immutable<CwuProposalCreate.State>;
+    cwuProposalEdit?: Immutable<CwuProposalEdit.State>;
+    cwuProposalView?: Immutable<CwuProposalView.State>;
+    cwuProposalList?: Immutable<CwuProposalList.State>;
   };
 }
 
+// Note(Jesse): @add_new_page_location
 type InnerMsg
   = ADT<'noop'>
   | ADT<'closeModal'>
-  | ADT<'nav', Nav.Msg>
-  | ADT<'pageLanding', PageLanding.Msg>
-  | ADT<'pageOpportunities', PageOpportunities.Msg>
-  | ADT<'pageContent', PageContent.Msg>
-  | ADT<'pageSignIn', PageSignIn.Msg>
-  | ADT<'pageSignOut', PageSignOut.Msg>
-  | ADT<'pageSignUpStepOne', PageSignUpStepOne.Msg>
-  | ADT<'pageSignUpStepTwo', PageSignUpStepTwo.Msg>
-  | ADT<'pageNotice', PageNotice.Msg>
-  | ADT<'pageUserList', PageUserList.Msg>
-  | ADT<'pageUserProfile', PageUserProfile.Msg>
-  | ADT<'pageOrgCreate', PageOrgCreate.Msg>
-  | ADT<'pageOrgList', PageOrgList.Msg>
-  | ADT<'pageOrgEdit', PageOrgEdit.Msg>;
+  | ADT<'nav',                   Nav.Msg>
+  | ADT<'pageLanding',           PageLanding.Msg>
+  | ADT<'pageOpportunities',     PageOpportunities.Msg>
+  | ADT<'pageContent',           PageContent.Msg>
+  | ADT<'pageSignIn',            PageSignIn.Msg>
+  | ADT<'pageSignOut',           PageSignOut.Msg>
+  | ADT<'pageSignUpStepOne',     PageSignUpStepOne.Msg>
+  | ADT<'pageSignUpStepTwo',     PageSignUpStepTwo.Msg>
+  | ADT<'pageNotice',            PageNotice.Msg>
+  | ADT<'pageUserList',          PageUserList.Msg>
+  | ADT<'pageUserProfile',       PageUserProfile.Msg>
+  | ADT<'pageOrgCreate',         PageOrgCreate.Msg>
+  | ADT<'pageOrgList',           PageOrgList.Msg>
+  | ADT<'pageOrgEdit',           PageOrgEdit.Msg>
+  | ADT<'cwuOpportunityCreate',  CwuOpportunityCreate.Msg>
+  | ADT<'cwuOpportunityEdit',    CwuOpportunityEdit.Msg>
+  | ADT<'cwuOpportunityView',    CwuOpportunityView.Msg>
+  | ADT<'cwuProposalCreate',     CwuProposalCreate.Msg>
+  | ADT<'cwuProposalEdit',       CwuProposalEdit.Msg>
+  | ADT<'cwuProposalView',       CwuProposalView.Msg>
+  | ADT<'cwuProposalList',       CwuProposalList.Msg>;
```

Comment:
All page messages should be prefixed with `page` to match the other page messages

Reply:
Change to `cwuProposalList` to `pageProposalList`